tests/Spec.hs

```diff
@@ -65,6 +65,13 @@ main = do
                     P.length tb1 `shouldBe` P.length tb2
                     zipWithM_ shouldBe (fmap fst tb2) (fmap fst tb1)
                     zipWithM_ shouldBe (fmap snd tb2) (fmap snd tb1)
+        describe "untar" $ do
+            around (withTempTarFiles baseTmp) $
+                it "create-intermediate" $ \(fpIn, hIn, outDir, fpOut) -> do
+                    hClose hIn
+                    extractTarball "tests/files/subdir.tar" (Just outDir)
```

Comment:
`extractTarballLenient` should be used here instead for two reasons:
* the attempt to restore ownership to `luispedro:luispedro` with `uid` and `gid`: `1001:1001` for the test file `subdir.tar` causes the tests on travis to fail (since such user is missing from there)
* test file is malformed (which is osk, since it is what we are trying to test here) and just as in the comment it should be restored leniently.